docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,71 @@
 
 With the Zowe Editor, you can create and edit the many types of files.
 
-## Specifying a language server
+## Specifying a highlighting language
 
-To specify a language server, complete these steps:
+1. Click **Language** on the editor toolbar. A dropdown menu will be displayed.
+2. From the dropdown, select the desired language. Plain Text will be chosen by default.  
 
-1. From the **Language Server** menu, select **URL***.
-2. From the **Language Server Setting**, **Put your config here** area, paste your configuration.
-3. Ensure that the **Enable Language Server** check box is selected.
-4. Click **Save**.
+## Open a dataset
 
-## Specifying a language
+To open a dataset, follow these steps:
 
-From the **Language** menu, select the language you want to use.
+1. From the **File** menu, select Open Datasets. You can also use (ALT+K).
+2. In the Dataset field, specify the name of the dataset you want to open. 
+3. Click **Open** 
+
+## Deleting a dataset
+
+To delete a dataset, follow these steps:
+
+1. In the file tree, right-click on a dataset you want to delete.
+2. From the right-click menu, click **Delete**. A warning dialogue will appear. 
+3. Click **Delete** 
 
 ## Opening a directory
 
 To open a directory on the system, complete these steps:
 
-1. From the **File** menu, select **Open Directory**.(Alternatively, you can click **Open Directory** in the File Explorer.)
-
-2. From the **Open Directory**, **Input Your Directory** field, type the name of the directory you want to open. For example: `/u/zs1234`
+1. From the **File** menu, select **Open Directory**. You can also use (ALT+O).
+2. In the Directory field, specify the name of the directory you want to open. For example: `/u/zs1234`
 3. Click **Open**.
```

Comment:
Here you have a period at the end of "Open" but don't have it in the other lines (create, save etc)